src/InvestmentManager.sol

```diff
@@ -294,7 +292,9 @@ contract InvestmentManager is Auth, IInvestmentManager {
         address vault = poolManager.getVault(poolId, trancheId, assetId);
 
         InvestmentState storage state = investments[vault][user];
-        require(state.exists == true, "InvestmentManager/non-existent-user");
+        require(
```

Comment:
The checks should be:
- in `fulfillDepositRequest`: `state.pendingDepositRequest > 0`
- in `fulfillRedeemRequest`: `state.pendingRedeemRequest > 0`
- in `fulfillCancelDepositRequest`: `state.pendingCancelDepositRequest == true`
- in `fulfillCancelRedeemRequest`: `state.pendingCancelRedeemRequest == true`